metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -154,54 +154,7 @@ def bigquery_audit_metadata_query_template(
 
         return textwrap.dedent(query)
 
-    def compute_bigquery_lineage_via_gcp_logging(
-        self, project_id: str
-    ) -> Dict[str, Set[LineageEdge]]:
-        logger.info(f"Populating lineage info via GCP audit logs for {project_id}")
-        try:
-            clients: GCPLoggingClient = _make_gcp_logging_client(project_id)
-
-            log_entries: Iterable[AuditLogEntry] = self._get_bigquery_log_entries(
-                clients
-            )
-            logger.info("Log Entries loaded")
-            parsed_entries: Iterable[QueryEvent] = self._parse_bigquery_log_entries(
-                log_entries
-            )
-            return self._create_lineage_map(parsed_entries)
-        except Exception as e:
-            self.error(
-                logger,
-                "lineage-gcp-logs",
-                f"Failed to get lineage gcp logging for {project_id}. The error message was {e}",
-            )
-            raise e
-
-    def compute_bigquery_lineage_via_exported_bigquery_audit_metadata(
-        self,
-    ) -> Dict[str, Set[LineageEdge]]:
-        logger.info("Populating lineage info via exported GCP audit logs")
-        try:
-            # For exported logs we want to submit queries with the credentials project_id.
-            _client: BigQueryClient = get_bigquery_client(self.config)
-            exported_bigquery_audit_metadata: Iterable[
-                BigQueryAuditMetadata
-            ] = self._get_exported_bigquery_audit_metadata(_client)
-            parsed_entries: Iterable[
-                QueryEvent
-            ] = self._parse_exported_bigquery_audit_metadata(
-                exported_bigquery_audit_metadata
-            )
-            return self._create_lineage_map(parsed_entries)
-        except Exception as e:
-            self.error(
-                logger,
-                "lineage-exported-gcp-audit-logs",
-                f"Error: {e}",
-            )
-            raise e
-
-    def compute_bigquery_lineage_via_catalog_lineage_api(
+    def lineage_via_catalog_lineage_api(
```

Comment:
Renamed `compute_bigquery_lineage_via_catalog_lineage_api` to be a bit more concise